openlibrary/templates/books/edition-sort.html

```diff
@@ -85,5 +85,13 @@
         <ul class="read">
           <li class="read-option">$:macros.LoanStatus(book)</li>
         </ul>
+        $if oclc_numbers or isbn:
+          <ul class="editions-table__world-cat-link">
```

Comment:
A `ul` element seems like an odd semantic choice here as nothing is being listed. Why not two spans?

Reply:
Side note: The BEM syntax is not right here as this is not a world-cat-link it's a list.